tests/quantization/ggml/test_ggml.py

```diff
@@ -303,6 +303,8 @@ class GgufModelTests(unittest.TestCase):
     gemma3_vision_model_id = "unsloth/gemma-3-4b-it-GGUF"
     qwen3_model_id = "Qwen/Qwen3-0.6B-GGUF"
     qwen3moe_model_id = "Qwen/Qwen3-30B-A3B-GGUF"
+    qwen2vl_model_id = "unsloth/Qwen2.5-VL-7B-Instruct-GGUF"
+    original_qwen2vl_model_id = "Qwen/Qwen2.5-VL-7B-Instruct"
```

Comment:
```suggestion
    qwen2vl_model_id = "unsloth/Qwen2.5-VL-3B-Instruct-GGUF"
    original_qwen2vl_model_id = "Qwen/Qwen2.5-VL-3B-Instruct"
```
I think we can use a smaller model for test.